basilisk/src/grid/gpu/Benchmarks.md

```
@@ -47,11 +46,13 @@ set title 'Speed in grid points x timesteps / second'
 plot '< awk -v findex=3 -v minlevel=6 -f advection.awk advection' using 2:xtic(1) ti col lc rgb c1, \
      '' u 3 ti col lc rgb c2, \
      '' u 4 ti col lc rgb c3, \
-     '' u 5 ti col lc rgb c4
+     '' u 5 ti col lc rgb c4, \
+	 '' u 6 ti col lc rgb c5
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)